src/ROUTES.ts

```diff
@@ -37,16 +36,9 @@ const ROUTES = {
     ALL_SETTINGS: 'all-settings',
 
     SEARCH: {
-        route: '/search/:query',
-        getRoute: (searchQuery: SearchQuery, queryParams?: AuthScreensParamList['Search_Central_Pane']) => {
-            const {sortBy, sortOrder} = queryParams ?? {};
-
-            if (!sortBy && !sortOrder) {
-                return `search/${searchQuery}` as const;
-            }
-
-            return `search/${searchQuery}?sortBy=${sortBy}&sortOrder=${sortOrder}` as const;
-        },
+        route: '/search',
+        getRoute: ({query, queryKind = CONST.SEARCH.QUERY_KIND.CANNED_QUERY, policyIDs}: {query: SearchQueryString; queryKind?: QueryKind; policyIDs?: string}) =>
+            `search?${queryKind === CONST.SEARCH.QUERY_KIND.CANNED_QUERY ? 'q' : 'cq'}=${query}${policyIDs ? `&policyIDs=${policyIDs}` : ''}` as const,
```

Comment:
I think the doc had [two separate routes](https://docs.google.com/document/d/1bRV1nPz8SaI_6DHqfehmLnADxv40us_q0JNvmZ1QyaY/edit#bookmark=id.888saqncimmk), any reason to combine them?

Reply:
I wonder if instead of updating `SearchPage` to also handle the Search Results case, we should just create two components for these pages, e.g. `SearchPage` and `SearchResultsPage`

Reply:
☝️ yes that is what @WojtekBoman is doing in his PR, Im assuming this code is temporary.

Reply:
If I understand correctly, the only difference between SearchPage and SearchResultsPage is the header on these pages. To me it's easier to display this header conditionally based on `SearchUtils.isCustomQuery` rather than creating two different pages and maintaining them separately. If we want to split this page into `SearchPage` and `SearchResultsPage`, we'll have to repeat huge amount of code only to display a different header. Maybe we can use the same page for now, and when these pages begin to differ significantly, we'll separate them. What do you think @luacmartins?

Reply:
That was the initial idea and what we have in the doc now. It seems like we started running into some issues with the `isCustomQuery` method, so that's why I suggested separating those screens. I'm cool with keeping them the same page for now. @adamgrzybowski what do you think?

Reply:
For our current requirements, I think one page is perfect. I already remove `cq` so now the params are 
`q`, `isCustomQuery` and optional `policyIDs`